oratorio/templates/coach/base.html

```diff
@@ -60,9 +60,11 @@
 			</a>
 		</li>
 		<li style="float:right">
-			<a href="profile">
-				User Profile
-			</a>
+      <div class="ProfileButton">
+  			<a>
```

Comment:
the <a>s are a bit unnecessary since you can make it appear stylistically the same as the others by just appending ProfileButton to the same rules but that might have unseen consequences and squash some behavior so.